test/dotnet-new.Tests/Approvals/DotnetNewDetailsTest.CanDisplayDetails_InstalledPackage_OtherFeed.verified.txt

```diff
@@ -1,41 +1,15 @@
-﻿Microsoft.Azure.WebJobs.ItemTemplates
-   Package version: 4.0.3038
-   Details: Collection of templates for Azure Functions
+﻿NUnit3.DotNetNew.Template
```

Comment:
Why did you switch which template we were downloading and how'd you pick this one?

Reply:
Thanks, I didn't notice that comment from before. I guess this one is more stable.